src/lib/components/Icon.svelte

```diff
@@ -79,7 +79,8 @@
   export let icon: IconType;
   export let width: string | number = "100%";
   export let height: string | number = "100%";
+  export let inline: boolean | number = false;
```

Comment:
Can this just be a boolean? In that case, the type annotation can actually be removed too:

```suggestion
  export let inline = false;
```